src/components/Table/index.js

```diff
@@ -353,13 +353,20 @@ const Table = () => {
           },
           {
             Header: () => {
+              const date = new Date(updated);
+              const month = date.toLocaleString('default', {
+                month: 'long',
+              });
+              const year = date.getFullYear();
               return (
                 <>
                   <div
                     style={{ whiteSpace: 'nowrap', display: 'inline-block' }}
                   >
                     Companies{' '}
-                    <span data-tip="Companies retrieved from Leetcode Premium (January 2022)">
+                    <span
+                      data-tip={`Companies retrieved from Leetcode Premium (${month} ${year})`}
```

Comment:
Thank you for this update - could we make a tiny adjustment?

```suggestion
                      data-tip={`Companies retrieved from Leetcode Premium on (${month} ${day}, ${year})`}
```

The list of companies would ideally be updated every week right after Leetcode update the companies for each question. It'd be nice to have the date included to understand the last time things were ran!